src/switch_core_media.c

```diff
@@ -10115,16 +10127,16 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
 
 
 	if (switch_channel_up(session->channel)) {
-		uint32_t codec_ms = a_engine->cur_payload_map->codec_ms * (flags[SWITCH_RTP_FLAG_USE_MILLISECONDS_PER_PACKET] ? 1 : 1000);
```

Comment:
This is needed. Make sure to use `codec_ms` here and replace `a_engine->cur_payload_map->codec_ms * 1000` in 10138.

Reply:
done. thanks!